entgql/template/pagination.tmpl

```diff
@@ -517,29 +517,21 @@ func ({{ $r }} *{{ $name }}) ToEdge(order *{{ $order }}) *{{ $edge }} {
 	if {{ $r }}, err = pager.applyFilter({{ $r }}); err != nil {
 		return nil, err
 	}
-	conn := &{{ $conn }}{Edges: []*{{ $edge }}{}}
```

Comment:
I think dropping the Edges causes it to marshal as `null` and not `[]`. Please verify this.